streamer/src/nonblocking/quic.rs

```diff
@@ -21,6 +25,8 @@ use {
     tokio::{task::JoinHandle, time::timeout},
 };
 
+const QUIC_TOTAL_STAKED_CONCURRENT_STREAMS: f64 = 100_000f64;
```

Comment:
How was this number chosen? With 383.3M SOL staked on mainnet, that means every 3833 SOL staked gives you a stream, which means that any node with less than that amount staked gets 0 streams and counts as unstaked.  According to Solana Beach, on mainnet, the 1776th staked node has just about enough stake for one stream, which means that ~90% of validators on the network get a stream (including a long tail of delinquent or totally unstaked validators).

So this number might even be a little too high, unless we know that a validator can handle a total of 100k concurrent streams.

Reply:
100k streams  * 1280 bytes per packet would be ~128MB and ~81MB for unstaked (128 * 500 * 1280) so that sounds pretty conservative in terms of packet memory use. Each stream and connection has other additional overhead as well though.